experimental/Schemes/src/ToricBlowups/attributes.jl

```diff
@@ -1,80 +1,77 @@
 export exceptional_prime_divisor
 
 @doc raw"""
-    underlying_morphism(bl::ToricBlowupMorphism)
+    underlying_morphism(phi::ToricBlowupMorphism) -> ToricMorphism
 
 Return the underlying toric morphism of a toric blowup. Access to other
 attributes such as `domain`, `codomain`, `covering_morphism` are
 executed via `underlying_morphism`.
 
 # Examples
 ```jldoctest
-julia> P3 = projective_space(NormalToricVariety, 3)
+julia> X = projective_space(NormalToricVariety, 3)
 Normal toric variety
 
-julia> f = blow_up(P3, [0, 1, 1])
+julia> phi = blow_up(X, [0, 1, 1])
 Toric blowup morphism
 
-julia> Oscar.underlying_morphism(f)
+julia> Oscar.underlying_morphism(phi)
 Toric morphism
 ```
 """
-underlying_morphism(bl::ToricBlowupMorphism) = bl.toric_morphism
-
-
+underlying_morphism(phi::ToricBlowupMorphism) = phi.toric_morphism
 
 @doc raw"""
-    index_of_exceptional_ray(bl::ToricBlowupMorphism)
+    index_of_exceptional_ray(phi::ToricBlowupMorphism) -> Int
 
 Return the index of the exceptional ray used in the construction of the toric
 blowup.
 
 # Examples
 ```jldoctest
-julia> P3 = projective_space(NormalToricVariety, 3)
+julia> X = projective_space(NormalToricVariety, 3)
 Normal toric variety
 
-julia> f = blow_up(P3, [0, 1, 1])
+julia> phi = blow_up(X, [0, 1, 1])
 Toric blowup morphism
 
-julia> index_of_exceptional_ray(f)
+julia> index_of_exceptional_ray(phi)
 5
 ```
 """
-index_of_exceptional_ray(bl::ToricBlowupMorphism) = bl.index_of_exceptional_ray
-
+index_of_exceptional_ray(phi::ToricBlowupMorphism) = phi.index_of_exceptional_ray
 
 @doc raw"""
-    minimal_supercone_coordinates_of_exceptional_ray(f::ToricBlowupMorphism) -> Vector{QQFieldElem}
+    minimal_supercone_coordinates_of_exceptional_ray(phi::ToricBlowupMorphism) -> Vector{QQFieldElem}
 
-Let $f\colon Y \to X$ be the toric blowup corresponding to a star
-subdivision along a ray with minimal generator $v$.
-This function returns the minimal supercone coordinate vector of $v$ in the fan of $X$.
+Let $\varphi\colon Y \to X$ be the toric blowup corresponding to a star
+subdivision along a ray with minimal generator $r$.
+This function returns the minimal supercone coordinate vector of $r$ in the fan of $X$.
 See `?minimal_supercone_coordinates` for more details.
```

Comment:
(I know this is not a change made in this PR, but I just noticed it.)

Why do we have this comment here? This seems a general feature of the REPL, and probably is to be advertise as such in the OSCAR general documention. Still, having this for individual docstrings feels wrong to me. My suggestion:
```suggestion
```

Reply:
You could instead do
```suggestion
See [`minimal_supercone_coordinates`](@ref) for more details.
```
This inserts a hyperlink to the documentation of minimal_supercone_coordinates.

Reply:
Removed the sentence and updated the docstring a bit.